dist/templates/ovnkube-node.yaml.j2

```diff
@@ -162,10 +163,10 @@ spec:
         securityContext:
           runAsUser: 0
           capabilities:
-            add: ["NET_ADMIN", "SYS_ADMIN", "SYS_PTRACE"]
-          {% if kind is defined and kind -%}
-          privileged: true
-          {% endif %}
+            add:
+            - NET_ADMIN
+            - SYS_PTRACE
```

Comment:
We need better documentation about why each capability is needed.
`SYS_PTRACE` was added in #842 but I don't understand why

Reply:
i think this is related to cni/shim execution context and SYS_PTRACE no longer will be needed once we have a configurable way of enabling unprivileged mode. Will re-test with removed SYS_PTRACE... and will get rid of it if not needed.